Eatery Blue/UI/EateryScreen/EateryPageViewController.swift

```diff
@@ -53,8 +54,10 @@ class EateryPageViewController: UIPageViewController {
             pages.append(eateryVC)
         }
         setViewControllers([pages[index]], direction: .forward, animated: true, completion: nil)
+        if let page = pages[index] as? EateryModelController{
```

Comment:
```suggestion
        if let page = pages[index] as? EateryModelController {
```
Missing a space before the {